miso-web/src/main/resources/ehcache.xml

```diff
@@ -70,10 +70,10 @@
       retryAttempts="0" retryAttemptDelaySeconds="0"/> -->
   </cache>
   
-  <cache name="targetedResequencingCache" eternal="true" overflowToDisk="true" statistics="true" maxBytesLocalHeap="30M" maxBytesLocalDisk="100M">
+  <cache name="targetedSequencingCache" eternal="true" overflowToDisk="true" statistics="true" maxBytesLocalHeap="30M" maxBytesLocalDisk="100M">
```

Comment:
These caches don't appear to be used at all, and they shouldn't be... Right? If you want to remove them, there are also caches defined for Plates in here still too